lib/parse.js

```diff
@@ -1,56 +1,58 @@
 const fury = require('fury');
 
+
 fury.use(require('fury-adapter-apib-parser'));
 fury.use(require('fury-adapter-swagger'));
 fury.use(require('fury-adapter-oas3-parser'));
 
-function createWarning(message) {
-  const annotationElement = new fury.minim.elements.Annotation(message);
-  annotationElement.classes.push('warning');
-  annotationElement.attributes.set('sourceMap', [
-    new fury.minim.elements.SourceMap([[0, 1]]),
+const { Annotation, SourceMap, ParseResult } = fury.minim.elements;
+
+
+function createAnnotation(type, message) {
+  const element = new Annotation(message);
+  element.classes.push(type);
+  element.attributes.set('sourceMap', [
+    new SourceMap([[0, 1]]),
```

Comment:
NOTE, this annotation doesn't contain line/column. Remember that case when implementing line/column logic from annotations.

Reply:
Yup, I know about it 👍